src/medicalpage/MedicationForm.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import SubmitButton from '../components/SubmitButton';
+import Input from '../components/Input';
+
+const uid = require('uid');
+
+const MedicationFormContainer = styled.form`
+  display: grid;
+`;
+const CreateMedicationEntryButton = styled.button`
+  grid-row: -1;
+  grid-column: 2 / span 4;
+  background: skyblue;
+`;
+
+export default function MedicationForm({ onSingleMedicationSubmit }) {
+  const [isFormVisible, setIsFormVisible] = useState(false);
+
+  function onSubmit(event, onSingleMedicationSubmit) {
```

Comment:
Does it make sense to extract this to a helper function? It feels defining and calling an `onSubmit` function within a component... 🤔